.env.example

```diff
@@ -0,0 +1,64 @@
+# ------------------------------
+# Environment Variables for server & worker
+# ------------------------------
+
+# ------------------------------
+# Server Configuration
+# ------------------------------
+
+# Log path
+LOG_PATH=/tmp
+# Server port
+SERVER_PORT=8080
+# Maximum number of workers
+WORKERS_MAX=100
+# Worker quit timeout in seconds
+WORKER_QUIT_TIMEOUT_SECONDES=60
+
+# Agora App ID and Agora App Certificate
+# required: this variable must be set
+AGORA_APP_ID=
+AGORA_APP_CERTIFICATE=
+
+# ------------------------------
+# Worker Configuration
+# ------------------------------
+
+# Extension: bedrock_llm
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_BEDROCK_MODEL=
+AWS_REGION=
+
+# Extension: agora_rtc
+# Azure STT key and region
+AZURE_STT_KEY=
+AZURE_STT_REGION=
+
+# Extension: azure_tts
+# Azure TTS key and region
+AZURE_TTS_KEY=
+AZURE_TTS_REGION=
+
+# Extension: cosy_tts
+# Cosy TTS key
+COSY_TTS_KEY=
+
+# Extension: elevenlabs_tts
+# ElevenLabs TTS key
+ELEVENLABS_TTS_KEY=
+
+# Extension: openai_chatgpt
+# OpenAI API key
+OPENAI_API_KEY=
+# OpenAI base URL
+# if using OpenAI, keep default. using other OpenAI-compatible providers, then set it to the other provider's address
+OPENAI_BASE_URL=
+# OpenAI Model
+OPENAI_MODEL=gpt-4o-mini
+# Proxy URL
+PROXY_URL=
```

Comment:
what kind of proxy url?

Reply:
[rename PROXY_URL to OPENAI_PROXY_URL](https://github.com/rte-design/ASTRA.ai/pull/147/commits/216efcaedba87c908c538ec8628e38fe38963ed1)